objective-c/HelloWorld/HelloWorld/AppDelegate.m

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.m
+//  HelloWorld
+//
+//  Created by Michael Lumish on 6/11/15.
+//  Copyright (c) 2015 Google. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+@interface AppDelegate ()
+
+@end
```

Comment:
This class extension is boilerplate that you aren't using; let's remove it to not confuse people.